README.md

```diff
@@ -1 +1,88 @@
-# wine_service
+# Weather Vine Wine Microservice
+
+## About this Project
+Weather Vine is an educational app for consumers to connect more deeply with the wine they enjoy. Explore wines from a region and see how the climate has influenced the very wine one drinks!   
+
+## Table of Contents
+
+  - [Getting Started](#getting-started)
+  - [Running the tests](#running-the-tests)
+  - [Service Oriented Architecture](#service-oriented-architecture)
+  - [Other Repos](#other-repos)
+  - [Database Schema](#database-schema)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [Versioning](#versioning)
+  - [Authors](#authors)
+  - [License](#license)
+  - [Acknowledgments](#acknowledgments)
+
+## Getting Started
+
+To get the web application running, please fork and clone down the repo.
+`git clone <your@github.account:WeatherVine/front_end.git>`
+
+### Prerequisites
+
+To run this application you will need Ruby 2.5.3 and Rails 5.2.5
+
+### Installing
+
+- Install the gem packages  
```

Comment:
With the `Figaro` gem, they may also need to run a `bundle exec figaro install` to generate the `application.yml` file so they can store the api key and then `.gitignore` that file